test/new-e2e/tests/windows/service-test/startstop_test.go

```diff
@@ -725,17 +728,18 @@ func (s *baseStartStopSuite) stopAllServices() {
 
 	// ensure all services are stopped
 	for _, serviceName := range s.getInstalledServices() {
-		s.Assert().EventuallyWithT(func(c *assert.CollectT) {
+		s.EventuallyWithExponentialBackoff(func() error {
 			status, err := windowsCommon.GetServiceStatus(host, serviceName)
-			if !assert.NoError(c, err) {
-				return
+			if err != nil {
+				return err
 			}
-			if !assert.Equal(c, "Stopped", status, "%s should be stopped", serviceName) {
-				s.T().Logf("%s still running, sending stop cmd", serviceName)
-				err := windowsCommon.StopService(host, serviceName)
-				assert.NoError(c, err, "should stop %s", serviceName)
+			if status != "Stopped" {
+				// In original code err := windowsCommon.StopService(host, serviceName)
+				// Not sure if it's a good idea to send a stop command at this point
+				return fmt.Errorf("%s should be stopped", serviceName)
```

Comment:
this `stopAllServices` function is just used between tests to quiesce the services. it's okay to keep sending stop, it's not meant to be part of the test.